Sources/BraintreePayPal/BTPayPalAppSwitchReturnURL.swift

```diff
@@ -11,10 +15,36 @@ enum BTPayPalAppSwitchReturnURLState {
 /// PayPal app switch authorization requests should result in success or user-initiated cancelation. These states are communicated in the url.
 struct BTPayPalAppSwitchReturnURL {
 
+    /// The overall status of the app switch - success, failure or cancelation
+    var state: BTPayPalAppSwitchReturnURLState = .unknown
+
+    /// The EC Token from the return URL.
+    var ecToken: String?
+
+    /// The BA Token from the return URL.
+    var baToken: String?
+
+    /// The timestamp from the return URL.
+    var timestamp: String?
```

Comment:
Similar comment here, we aren't using this.

This surfaces that we need to ask the analytics folks what they want us to do with this number, let's ask them! They may want us including it in an additional analytic event send